make/data/fontconfig/windows.fontconfig.properties

```diff
@@ -144,56 +145,56 @@ dialog.plain.chinese-ms950=MingLiU
 dialog.plain.chinese-ms950-extb=MingLiU-ExtB
 dialog.plain.hebrew=David
 dialog.plain.japanese=MS Gothic
-dialog.plain.korean=Gulim
+dialog.plain.korean=Malgun Gothic
 
 dialog.bold.alphabetic=Arial Bold
 dialog.bold.chinese-ms950=PMingLiU
 dialog.bold.chinese-ms950-extb=PMingLiU-ExtB
 dialog.bold.hebrew=David Bold
 dialog.bold.japanese=MS Gothic
-dialog.bold.korean=Gulim
+dialog.bold.korean=Malgun Gothic Bold
 
 dialog.italic.alphabetic=Arial Italic
 dialog.italic.chinese-ms950=PMingLiU
 dialog.italic.chinese-ms950-extb=PMingLiU-ExtB
 dialog.italic.hebrew=David
 dialog.italic.japanese=MS Gothic
-dialog.italic.korean=Gulim
+dialog.italic.korean=Malgun Gothic
 
 dialog.bolditalic.alphabetic=Arial Bold Italic
 dialog.bolditalic.chinese-ms950=PMingLiU
 dialog.bolditalic.chinese-ms950-extb=PMingLiU-ExtB
 dialog.bolditalic.hebrew=David Bold
 dialog.bolditalic.japanese=MS Gothic
-dialog.bolditalic.korean=Gulim
+dialog.bolditalic.korean=Malgun Gothic
```

Comment:
You should use the Bold version of the font here too - ie for bold italic - not just for bold - since it is clearly closer ..